test/common/include/alpaka/test/acc/TestAccs.hpp

```diff
@@ -83,15 +83,14 @@ namespace alpaka
 #if defined(ALPAKA_ACC_ANY_BT_OMP5_ENABLED) && !defined(TEST_UNIT_KERNEL_KERNEL_STD_FUNCTION)                         \
     && !(                                                                                                             \
         BOOST_COMP_GNUC                                                                                               \
-        && (((BOOST_COMP_GNUC < BOOST_VERSION_NUMBER(11, 0, 0))                                                       \
-                 && /* tests excluded because of GCC10 Oacc / Omp5 target symbol bug with multiple units */           \
-                 defined(TEST_UNIT_BLOCK_SHARED)                                                                      \
-             || defined(TEST_UNIT_BLOCK_SYNC) || defined(TEST_UNIT_WARP) || defined(TEST_UNIT_INTRINSIC)              \
-             || defined(TEST_UNIT_KERNEL) || defined(TEST_UNIT_MEM_VIEW))                                             \
+        && (((BOOST_COMP_GNUC < BOOST_VERSION_NUMBER(11, 0, 0)) /* tests excluded because of GCC10 Oacc / Omp5 target \
```

Comment:
clang-format forces this longer inline comment to the end of the line an aligns it. This looks strange.

Reply:
Maybe put it above the `&&`? Like so:

```c++
BOOST_COMP_GNUC
 /* tests excluded because of GCC10 Oacc / Omp5 target symbol bug with multiple units */
&& (((BOOST_COMP_GNUC < BOOST_VERSION_NUMBER(11, 0, 0))
```

Reply:
It does not belong before the `&&`, because that first condition, which it refers to, is for the GCC 10 issue, there is a second condition which is for GCC in general, separate from the "multiple units"-bug referred to in the comment.

If put in the place where it is in the old version of the code, clang-format would evidently not move it, as this already went through clang format. I did not want to leave it after the following `&&`, because placing it there may have let to forgetting the parenthesis in the first place.

Reply:
Btw.: I think in your example the comment would be put after `BOOST_COMP_GNUC`, which would look less bad but would be more unsuitable.

Reply:
I can either
1. leave it as it is now (in which case this PR can be merged), or
2. return the comment to it former place (old state in diff).

How do people feel about this?